src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -6,18 +6,12 @@
 from knack.util import CLIError
 from knack.log import get_logger
 
-from azure.cli.core.commands import LongRunningOperation
-
-from ._constants import get_classic_sku, get_managed_sku, get_premium_sku
+from ._constants import get_managed_sku, get_premium_sku
 from ._utils import (
-    arm_deploy_template_new_storage,
-    arm_deploy_template_existing_storage,
-    random_storage_account_name,
     get_registry_by_name,
     validate_managed_registry,
     validate_sku_update,
-    get_resource_group_name_by_registry_name,
-    get_resource_id_by_storage_account_name
```

Comment:
Can this method be removed from utils?

Reply:
I think everything about storage can be removed

Reply:
Removed.